app/src/main/java/org/fossasia/susi/ai/device/DevicePresenter.kt

```diff
@@ -115,7 +116,47 @@ class DevicePresenter(deviceActivity: DeviceActivity, manager: WifiManager) : ID
         }
 
         override fun onPostExecute(result: Void?) {
-            deviceView?.onDeviceConnectionSuccess()
+            Timber.d("Connected")
+
         }
     }
+
+    override fun makeConnectionRequest() {
+        Timber.d("make request")
+        deviceView?.unregister()
+        // test data only
```

Comment:
this data is only for @stealthanthrax as only he can test the app , this will be removed further but right now it is kept for his reference